From 7e3e1b7a39384b5fc06ee781e1ef8731b7f820c0 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Wed, 12 Aug 2015 21:55:08 +0800
Subject: [PATCH 0224/5242] MLK-11343-02 cpufreq: imx: add more clk used by
 cpufreq

commit  eb43bdc53b94146870ba3e846a307cda58f29f1c from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds pll1, pll_bypass and pll1_bypass_src that
will be used in ARM clock switching code.

Signed-off-by: Bai Ping <b51503@freescale.com>
Updated during 4.14 rebase
Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/cpufreq/imx6q-cpufreq.c |   10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/drivers/cpufreq/imx6q-cpufreq.c b/drivers/cpufreq/imx6q-cpufreq.c
index c068acc..0ed0c72 100644
--- a/drivers/cpufreq/imx6q-cpufreq.c
+++ b/drivers/cpufreq/imx6q-cpufreq.c
@@ -31,12 +31,15 @@ enum IMX6_CPUFREQ_CLKS {
 	STEP,
 	PLL1_SW,
 	PLL2_PFD2_396M,
+	PLL1,
+	PLL1_BYPASS,
+	PLL1_BYPASS_SRC,
 	/* MX6UL requires two more clks */
 	PLL2_BUS,
 	SECONDARY_SEL,
 };
-#define IMX6Q_CPUFREQ_CLK_NUM		5
-#define IMX6UL_CPUFREQ_CLK_NUM		7
+#define IMX6Q_CPUFREQ_CLK_NUM		8
+#define IMX6UL_CPUFREQ_CLK_NUM		10
 
 static int num_clks;
 static struct clk_bulk_data clks[] = {
@@ -45,6 +48,9 @@ enum IMX6_CPUFREQ_CLKS {
 	{ .id = "step" },
 	{ .id = "pll1_sw" },
 	{ .id = "pll2_pfd2_396m" },
+	{ .id = "pll1" },
+	{ .id = "pll1_bypass" },
+	{ .id = "pll1_bypass_src" },
 	{ .id = "pll2_bus" },
 	{ .id = "secondary_sel" },
 };
-- 
1.7.9.5

